app/javascript/dashboard/i18n/locale/en/conversation.json

```diff
@@ -41,6 +41,7 @@
     "SAVE_CONTACT": "Save",
     "UPLOADING_ATTACHMENTS": "Uploading attachments...",
     "REPLIED_TO_STORY": "Replied to your story",
+    "UNSUPPORTED_MESSAGE": "This message is unsupported. You can view this message on the mobile app.",
```

Comment:
lets also tweak this text little bit.  view this on mobile app must be confusing to the user as they would imagine the mobile app to be chatwoot. lets change this to reflect that they need to view this message on facbook/instagram